pkg/logs/sds/scanner.go

```diff
@@ -23,6 +25,15 @@ const ScannedTag = "sds_agent:true"
 
 const SDSEnabled = true
 
+var (
+	tlmSDSRulesState = telemetry.NewGaugeWithOpts("sds", "rules", []string{"pipeline", "state"},
+		"Rules state.", telemetry.Options{DefaultMetric: true})
+	tlmSDSReconfigError = telemetry.NewCounterWithOpts("sds", "reconfiguration_error", []string{"pipeline", "type", "error_type"},
+		"Count of SDS reconfiguration error.", telemetry.Options{DefaultMetric: true})
+	tlmSDSReconfigSuccess = telemetry.NewCounterWithOpts("sds", "reconfiguration_success", []string{"pipeline", "type"},
+		"Count of SDS reconfiguration success.", telemetry.Options{DefaultMetric: true})
```

Comment:
One other question about these new metrics: do we need to be storing metadata for them somewhere so that the app sets the correct units and metric type?